docs/setuptools.rst

```diff
@@ -126,14 +126,15 @@ These would be the modified contents of ``setup.py``::
 
     setup(
         name='yourpackage',
-        version='0.1',
+        version='0.1.0',
```

Comment:
please don't make unrelated changes

Reply:
Really? Quite common? Not with semver.

You're semi-right: It's a code snippet. I'm not changing the behavior of the application. I'm adapting the code snippet.

Reply:
BTW, I would also try to fix #1239 with this PR. I think it makes sense to do it in one go. Still a code snippet change.

Reply:
0.1 and 0.1.0 have the same meaning even with semver, but i just checked the spec and they indeed expect you to always put x.y.z and not omit z if z==0...

Reply:
Thank you for verifying. :+1: So, it's okay that I leave it in there?

Reply:
Ideally it should be in a separate commit (it's a different change), but the change is trivial so I'll just go ahead and merge it as it is

Reply:
Click has always been 2 numbers, not 3. Please revert this.

Reply:
@davidism it's not the click version but the example for people's own packages - otherwise i wouldn't have accepted it

Reply:
Yeah, I noticed right after, never mind.